diff --git a/findutils.old/gnulib/lib/freadahead.c b/findutils.new/gnulib/lib/freadahead.c
index daad290..2a18e74 100644
--- a/findutils.old/gnulib/lib/freadahead.c
+++ b/findutils.new/gnulib/lib/freadahead.c
@@ -60,7 +60,5 @@ freadahead (FILE *fp)
     return 0;
   /* fp->_Buf <= fp->_Next <= fp->_Rend */
   return fp->_Rend - fp->_Next;
-#else
- #error "Please port gnulib freadahead.c to your platform! Look at the definition of fflush, fread on your system, then report this to bug-gnulib."
 #endif
 }
